components/elm/src/external_models/emi/src/elm_stub/utils/clm_varctl.F90

```diff
@@ -333,7 +333,7 @@ module clm_varctl
   !----------------------------------------------------------
```

Comment:
Should the filename be renamed?

Reply:
Note that this PR is focused on *variables* .  Comment strings can be done later.

Reply:
Thanks for letting me know. @bishtgautam please ignore my comments related.

Reply:
Opened a new issue https://github.com/E3SM-Project/E3SM/issues/3972 for it.